### PR TITLE
Add changelog for v1.8.3

### PR DESCRIPTION
**Pull request by MarcosDY**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## [1.8.2] - 2023-10-25
+
+### Added
+
+- The spire-agent has a more uniform distribution of SVID renewal requests to spire-server (#4534)
```
Comment (azdagron) [inline CHANGELOG.md:7]:
```suggestion
- SPIRE Agent distributes sync requests to the SPIRE server to mitigate thundering herd situations (#4534)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+- Documentation improvements (#4579, #4569)
+
+### Changed
+
+- The Spire agent now retries to sync on startup (#4479)
```
Comment (azdagron) [inline CHANGELOG.md:12]:
```suggestion
- SPIRE agent performs the initial sync more aggressively when tuned with a longer sync interval (#4479)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+
+### Fixed
+
+- Release artifacts have the correct version information (#4564)
+- The spire agent to prevent to use `insecureBootstrap` and `trustBundleUrl` at the same time (#4532)
```
Comment (azdagron) [inline CHANGELOG.md:17]:
```suggestion
- The SPIRE Agent `insecureBootstrap` and `trustBundleUrl` configurables are now mutually exclusive (#4532)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+- Credential composer can parse claims (#4489)
```
Comment (azdagron) [inline CHANGELOG.md:18]:
```suggestion
- Bug preventing JWT-SVIDs from being minted when a Credential Composer plugin is configured (#4489)
```

**Review by azdagron — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## [1.8.2] - 2023-10-25
```
Comment (rturner3) [inline CHANGELOG.md:3]:
```suggestion
## [1.8.3] - 2023-10-25
```

**Review by rturner3 — CHANGES_REQUESTED** [review]
The contents of the changelog entry look good to me, but the version number needs to be incremented.

**Comment by amartinezfayo** [comment]
https://github.com/spiffe/spire/pull/4535 finally landed in 1.8.3 and should be included.

**Review by rturner3 — APPROVED** [review]

**Review by amartinezfayo — APPROVED** [review]
Thank you @MarcosDY!

**Pull request merged**